test-network-nano-bash/README.md

```diff
@@ -120,7 +131,7 @@ export CHAINCODE_SERVER_ADDRESS=127.0.0.1:9999
 And start the chaincode service:
 
 ```
-npm run start:server-notls
+npm run start:server-nontls
```

Comment:
is it a typo?

Reply:
Hi @timo-kang, the correct script is [start:server-nontls](https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-basic/chaincode-typescript/package.json#L21) so there is currently a typo in the test-network-nano-bash readme. I included the fix in this draft PR but it might be better to fix sooner since it's not really related to the other changes here.